scripts/process_cams_to_csv.py

```
@@ -0,0 +1,253 @@
+        print(f"Records: {len(df)}")
+        print(f"Time range: {df['timestamp'].min()} to {df['timestamp'].max()}")
+        
+        if 'pm25_ugm3_mean' in df.columns:
+            col = 'pm25_ugm3_mean'
+        elif 'pm25_ugm3' in df.columns:
+            col = 'pm25_ugm3'
+        else:
+            col = None
+            
+        if col:
+            print(f"\nPM2.5 Statistics (µg/m³):")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)